test('Hello word', () => {

})

test('Test failure', () => {
    throw new Error('Test failure!!!')
})